packages/blocks/src/__internal__/rich-text/rich-text-operations.ts

```diff
@@ -542,6 +543,28 @@ export function handleLineStartBackspace(page: Page, model: ExtendedModel) {
   handleUnknownBlockBackspace(model);
 }
 
+function handleParagraphBlockLeftKey(page: Page, model: ExtendedModel) {
+  if (!matchFlavours(model, ['affine:paragraph'])) return false;
+  const pageElement = getDefaultPage(page);
+  if (!pageElement) {
+    // Maybe in edgeless mode
+    return null;
+  }
+  const titleVEditor = pageElement.titleVEditor;
+  const previousBlock = getPreviousBlock(model);
+  if (previousBlock !== null) {
+    if (matchFlavours(previousBlock, ['affine:surface'])) {
```

Comment:
I feel that the way judgments are made here is somewhat unreliable. Do you have a better approach to handle it? @Saul-Mirone